db/seeds.rb

```diff
@@ -152,3 +202,15 @@
     phone: '012-345-6789', reservations: r_and_d.sample, delivery: r_and_d.sample
     )
 end
+
+user = User.new(email: "seed@seeder.com", password: "seedpassword")
+user.save!
+
+Restaurant.all.each do |restaurant|
```

Comment:
this is a scalability concern. If I had a million Restaurants I would load them all into memory here. Check out the `find_each` class method